apps/staking/src/components/AccountSummary/index.tsx

```diff
@@ -130,16 +132,25 @@ export const AccountSummary = ({
               actionDescription="Move funds from your account back to your wallet"
               actionName="Withdraw"
               max={availableToWithdraw}
-              transfer={withdraw}
-              isDisabled={availableToWithdraw === 0n}
+              {...(api.type === ApiStateType.Loaded && {
+                transfer: api.withdraw,
+              })}
```

Comment:
Note to reviewer: in this PR I made the `<TransferButton>` component render a disabled button if the max is `0n` or if `transfer` is undefined.  That makes it a lot easier to just use it all over and let it disable itself correctly if there's no function to call (generally meaning the API is in a state where this transfer is impossible) or if there's no tokens to transfer.  So you'll see here and a few other places where I removed some checks that control the button states, since those are now handled by the button itself.